dev/com.ibm.ws.jaxws.ejb_fat/fat/src/com/ibm/ws/jaxws/ejb/suite/FATSuite.java

```diff
@@ -42,4 +46,7 @@
                 EJBWSInterceptorTest.class
 })
 public class FATSuite {
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification().andWith(new FeatureReplacementAction().addFeature("jaxws-2.3").removeFeature("jaxws-2.2").removeFeature("jsp-2.2").removeFeature("servlet-3.1").removeFeature("jaxwstest-2.2").withID("jaxws-2.3"));
```

Comment:
jsp-2.2, servlet-3.1 and jaxwstest-2.2 are not necessary since server.xml files don't use them in this fat project.

Reply:
Removed